### PR TITLE
Kafka topic minio events

### LINKED ISSUE #4: Give the new arborescence of /outputs/<activityid>/traces.json

### LINKED ISSUE #5: When a new file is detected in minio via kafka topic, trace allocator should be able to compact it

### PR DESCRIPTION
**Pull request by jujusb**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,10 +14,42 @@ const MAX_WAIT_TIME_ON_EXIT = 30*1000;
...
+            intervalId = setInterval(run, config.refreshInterval);
+        } else {
+            await run();
+        }
+
+        logger.info('Compactor initialized.');
+    } catch (error) {
+        logger.error('Error during compactor initialization:', error);
+    }
 }
```
Comment (imartinezortiz) [inline src/index.js]: Define two functions with the actual code and call them here

File: src/kafka.js
Comment (imartinezortiz) [inline src/kafka.js:1]: Test / check if the kafka connector retries the connection if lost / closed.

**Review by imartinezortiz — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (jujusb) [reply src/index.js]: fixed

File: src/kafka.js
Comment (jujusb) [reply src/kafka.js:1]: retry automatically when connection lost / closed

**Review by imartinezortiz — APPROVED** [review]

**Pull request merged**